.gitignore

```diff
@@ -10,7 +10,6 @@
 packetbeat.test
 *.dev.yml
 
```

Comment:
For my understanding, is `.gitignore` simply copied over from libbeat? We should maybe put a warning comment not to change the ones from the Beats directly.